<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmFitCircleClass</class>
 <widget class="QDialog" name="frmFitCircleClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>655</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1160</width>
    <height>655</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1160</width>
    <height>655</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>拟合圆</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(220, 220, 220);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>550</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #bcc2bc;</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>基本设置</string>
      </attribute>
      <widget class="QGroupBox" name="groupCamera">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>18</y>
         <width>427</width>
         <height>73</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dfd5cf, stop: 0.15 #acbbac, stop: 0.85 #acbbac,stop:1 #dfd5cf);
color: rgb(85, 0, 255);</string>
       </property>
       <property name="title">
        <string>图像对象</string>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>42</y>
          <width>79</width>
          <height>19</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(30, 30, 30);
background: transparent;</string>
        </property>
        <property name="text">
         <string>输入图像：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtLinkImage">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>39</y>
          <width>255</width>
          <height>27</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 200);
background-color: rgba(170, 170, 127,50);</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="btnLinkImage">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>39</y>
          <width>27</width>
          <height>27</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>链接图像</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget{ border:none;border-radius: 8px;padding: 2px 4px;}
QPushButton:hover{ background-color: rgba(85, 150, 127, 100);}
QPushButton:pressed{background-color: rgba(85, 180, 127, 160);}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="sub_res.qrc">
          <normaloff>:/resource/link.png</normaloff>:/resource/link.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>27</width>
          <height>27</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="btnDelLinkImage">
        <property name="geometry">
         <rect>
          <x>394</x>
          <y>39</y>
          <width>27</width>
          <height>27</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>删除图像链接</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget{ border:none;border-radius: 8px;padding: 2px 4px;}
QPushButton:hover{ background-color: rgba(226, 142, 162, 100);}
QPushButton:pressed{background-color: rgba(226, 172, 162, 200);}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="sub_res.qrc">
          <normaloff>:/resource/del_link.png</normaloff>:/resource/del_link.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>27</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupC2">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>108</y>
         <width>427</width>
         <height>72</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dfd5cf, stop: 0.15 #acbbac, stop: 0.85 #acbbac,stop:1 #dfd5cf);
color: rgb(85, 0, 255);</string>
       </property>
       <property name="title">
        <string>构建参数</string>
       </property>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>42</y>
          <width>79</width>
          <height>19</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(30, 30, 30);
background: transparent;</string>
        </property>
        <property name="text">
         <string>轮廓点集：</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnDelLinkContour">
        <property name="geometry">
         <rect>
          <x>394</x>
          <y>38</y>
          <width>27</width>
          <height>27</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>删除轮廓点集链接</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget{ border:none;border-radius: 8px;padding: 2px 4px;}
QPushButton:hover{ background-color: rgba(226, 142, 162, 100);}
QPushButton:pressed{background-color: rgba(226, 172, 162, 200);}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="sub_res.qrc">
          <normaloff>:/resource/del_link.png</normaloff>:/resource/del_link.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>27</width>
          <height>27</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="btnLinkContour">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>38</y>
          <width>27</width>
          <height>27</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>链接轮廓点集</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget{ border:none;border-radius: 8px;padding: 2px 4px;}
QPushButton:hover{ background-color: rgba(85, 150, 127, 100);}
QPushButton:pressed{background-color: rgba(85, 180, 127, 160);}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="sub_res.qrc">
          <normaloff>:/resource/link.png</normaloff>:/resource/link.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>27</width>
          <height>27</height>
         </size>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtLinkContour">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>38</y>
          <width>255</width>
          <height>27</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>vector&lt;cv::Point2f&gt;类型</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 200);
background-color: rgba(170, 170, 127,50);</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupC3">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>197</y>
         <width>427</width>
         <height>320</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dfd5cf, stop: 0.15 #acbbac, stop: 0.85 #acbbac,stop:1 #dfd5cf);
color: rgb(85, 0, 255);</string>
       </property>
       <property name="title">
        <string>输出数据(圆的中心坐标、半径)</string>
       </property>
       <widget class="QTextEdit" name="txtMsg">
        <property name="geometry">
         <rect>
          <x>4</x>
          <y>32</y>
          <width>419</width>
          <height>283</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QTextEdit         
{
    background-color: rgba(180, 180, 215,120);    
    gridline-color: rgba(220, 220, 255,255);
    color: rgba(20, 20, 20,200);
}
QScrollBar:vertical{ 
		background:#FFFFF2;
		padding-top:20px;
		padding-bottom:20px;
		padding-left:2px;
		padding-right:2px;
		border-left:1px solid #d7d7d7;}
QScrollBar::handle:vertical{
		background:#dbdbdb;
		border-radius:6px;
		min-height:20px;}
QScrollBar::handle:vertical:hover{
		background:#d0d0d0;}
QScrollBar::add-line:vertical{
		background:url(:/resource/down.png) center no-repeat;}
QScrollBar::sub-line:vertical{
		background:url(:/resource/up.png) center no-repeat;}
QScrollBar:horizontal{ 
		background:#FFFFF2;
		padding-top:20px;
		padding-bottom:20px;
		padding-left:2px;
		padding-right:2px;
		border-left:1px solid #d7d7d7;}
QScrollBar::handle:horizontal{
		background:#dbdbdb;
		border-radius:6px;
		min-height:20px;}
QScrollBar::handle:horizontal:hover{
		background:#d0d0d0;}
QScrollBar::add-line:horizontal{
		background:url(:/resource/right.png) center no-repeat;}
QScrollBar::sub-line:horizontal{
		background:url(:/resource/left.png) center no-repeat;}</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <layout class="QGridLayout" name="imageLayout">
     <property name="spacing">
      <number>1</number>
     </property>
    </layout>
   </item>
   <item row="0" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>42</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>42</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>83</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>单元描述：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtDescribe">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>357</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnExecute">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>37</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget{ background-color: rgb(85, 135, 127);}
QWidget{ color: rgb(235, 235, 235);}
QWidget{ border: 2px groove gray;border-radius: 8px;padding: 2px 4px;}
QPushButton:hover{ background-color: rgb(85, 155, 127);}
QPushButton:pressed{background-color: rgb(85, 175, 127);}</string>
       </property>
       <property name="text">
        <string>执行</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>6</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>3</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sub_res.qrc"/>
  <include location="sub_res.qrc"/>
  <include location="../FitEllipse/sub_res.qrc"/>
 </resources>
 <connections/>
</ui>
